it-test/src/test/resources/docker-compose.yaml

```diff
@@ -25,17 +25,17 @@ services:
       retries: 3
 
   horizon:
-    image: ${DOCKER_REGISTRY:-docker.io}/${DOCKER_ORG:-opennms}/${HORIZON_OCI:-horizon}:${ONMS_VERSION:-release-30.x}
+    image: ${DOCKER_REGISTRY:-docker.io}/${DOCKER_ORG:-opennms}/${HORIZON_OCI:-horizon}:${ONMS_VERSION:-30.0.3}
     hostname: horizon
     sysctls:
       net.ipv4.ping_group_range: "0 429496729"
     depends_on:
       - database
     environment:
       TZ: ${TIMEZONE:-America/New_York}
-    volumes:
+   # volumes:
       # we overlay the deploy folder with kar. It contains the plugin jars and dependencies:
-      - ${USER_HOME}/.m2/repository/org/opennms/plugins/cloud/kar/1.0.0-SNAPSHOT:/usr/share/opennms/.m2/repository/org/opennms/plugins/cloud/kar/1.0.0-SNAPSHOT:ro
```

Comment:
USER_HOME is set here: 
https://github.com/OpenNMS/opennms-cloud-plugin/blob/jira/DC-316/it-test/src/test/java/org/opennms/plugins/cloud/tsaas/TsaasStorageIT.java#L37
But maybe we don't need that